shiny/api-examples/data_frame/app-core.py

```diff
@@ -44,9 +44,9 @@ def summary_data():
 
     @reactive.calc
     def filtered_df():
-        # input.summary_data_selected_rows() is a tuple, so we must convert it to list,
+        # summary_data.selected_rows() is a tuple, so we must convert it to list,
         # as that's what Pandas requires for indexing.
-        selected_idx = list(req(input.summary_data_selected_rows()))
```

Comment:
One issue here is that this does introduce some inconsistency with the plot brushing options, maybe we should hold off changing the examples until we wrap those functions in methods.

Reply:
Just that for plot brushing we still have the `input.<id>_brushed_points` rather than using the object. I'm probably overthinking this.

Reply:
Yes, we haven't implemented the plot brushing yet. 

I'm happy that we're exploring this approach and the changes are good.